### DISCUSSION TITLE
Problem with behaviour not displaying

### DISCUSSION THREAD

**Discussion by ghost** [bridge-core/bridge.#33] [Q&A] [answered]
Every time i'm done with a behaviour pack it mainly put it in to development_behaviour_pack but it doesn't display on MC

**Comment by solvedDev** [comment, marked as answer]
Behavior packs belong into the "development_behavior_packs" folder while developing them and Minecraft even offers some usability advantages if you put your packs in this folder. If your pack doesn't load, your manifest.json file is probably invalid or your folder structure is wrong. Please send me a download link for a pack that is not loading in-game so I can investigate what you're doing wrong.
